Cargo.toml

```diff
@@ -5,3 +5,7 @@ members = [
     "saphir_macro",
     "saphir_cli",
 ]
+
+[profile.cli-release]
+inherits = "release"
+lto = "fat"
```

Comment:
It's not too long to build the saphir_cli like this, and it improves openapi generation speed a little bit.